ansible_base/rest_filters/apps.py

```diff
@@ -0,0 +1,8 @@
+from django.apps import AppConfig
+
+import ansible_base.lib.checks  # noqa: F401 - register checks
+
+
+class AuthenticationConfig(AppConfig):
+    default_auto_field = 'django.db.models.BigAutoField'
+    name = 'ansible_base.rest_filters'
```

Comment:
Why do filters need to be a separate app? There aren't any migrations, views, or URLs here.

Reply:
To be formal about it - (4)'s point is that it needs some default values (or modifications to dicts) to settings for the feature to function. Users could apply the needed settings on their own, and if they do, they don't need to add it to INSTALLED_APPS for it to work.

I see John's vision of management of features, and I appreciate the consistency it offers. And if people don't like it, I don't think it hurts anything to have these as apps.